src/main/resources/logback-spring.xml

```diff
@@ -29,19 +29,19 @@
     </logger>
 
```

Comment:
I think we also want to remove these sections
```xml
<springProfile name="dev || stdout">
<springProfile name="!(dev || stdout)">
```

For reference see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/8c18ec06ac71b8e3bb97c1c2176d441170db2e97/src/main/resources/logback-spring.xml

Reply:
Done ✅